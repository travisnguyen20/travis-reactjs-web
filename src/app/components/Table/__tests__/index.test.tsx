import * as React from 'react';
import { render } from '@testing-library/react';

import { Table } from '..';

describe('<Table  />', () => {
  it('should match snapshot', () => {
    const loadingIndicator = render(<Table columns={[]} data={[]} />);
    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
