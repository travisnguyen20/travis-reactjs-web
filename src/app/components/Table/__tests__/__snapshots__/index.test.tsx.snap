// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Table  /> should match snapshot 1`] = `
<table
  class="table"
  role="table"
>
  <thead />
  <tbody
    role="rowgroup"
  />
</table>
`;
